apps/btshell/src/cmd.c

```diff
@@ -3540,6 +3543,24 @@ static const struct shell_cmd_help l2cap_disconnect_help = {
     .params = l2cap_disconnect_params,
 };
 
+/*****************************************************************************
+ * $l2cap-reconfig                                                           *
+ *****************************************************************************/
+
+static const struct shell_param l2cap_reconfig_params[] = {
+    {"conn", "connection handle, usage: =<UINT16>"},
+    {"mtu", "new mtu, usage: =<UINT16>, default: 0 (no change)"},
+    {"mps", "new mps, usage: =<UINT16>, default: 0 (no change)"},
+    {"idxs", "list of channel indexes, usage: idx_1:idx_2:...:idx_5"},
```

Comment:
It wasn't clear for me at first how to use this parameter, maybe it could be improved.
Example usage: idxs=00:11